test/reference.html

```diff
@@ -0,0 +1,946 @@
+<!DOCTYPE html><html><head>
+<title>MathQuill Reference</title>
+<link rel="stylesheet" type="text/css" href="../build/mathquill.css">
+<style type="text/css">
+
+body {
+    margin-left: auto;
+    margin-right: auto;
+    width: 1280px;
+    max-width: 1280px;
+}
+
+p {
+    max-width: 700px;
+}
+
+h2 {
+    font-variant: small-caps;
+}
+
+h3 {
+    font-variant: small-caps;
+
+}
+
+code {
+    font-family: monospace;
+    font-size: 1.2em;
+}
+
+table tr {
+    -webkit-column-break-inside: avoid;
+    -moz-column-break-inside: avoid;
+    break-inside: avoid;
+}
+
+tr > td:first-child {
+    text-align: right;
+    padding-right: .5em;
+    display: table-cell; /* mq-math-mode sets the display to inline-block. Need override for proper alignment.*/
+}
+
+tr > td:nth-child(2) {
+    text-align: right;
+}
+
+
+.col2 {
+    -webkit-column-count: 2;
+    -moz-column-count: 2;
+    column-count: 2;
+}
+
+.col3 {
+    -webkit-column-count: 3;
+    -moz-column-count: 3;
+    column-count: 3;
+}
+
+.col4 {
+    -webkit-column-count: 4;
+    -moz-column-count: 4;
+    column-count: 4;
+}
+
+.col5 {
+    -webkit-column-count: 5;
+    -moz-column-count: 5;
+    column-count: 5;
+}
+
+img {
+    opacity: .5;
```

Comment:
The opacity? I was trying to find some way to make the codecogs rendering less prominent. I've removed the opacity trick and just change the background color now.